openapi-contracts/src/main/com/sinch/sdk/domains/verification/models/v1/webhooks/FlashCallRequestEventResponseFlashCallContent.java

```diff
@@ -19,20 +19,20 @@
 public interface FlashCallRequestEventResponseFlashCallContent {
 
   /**
-   * The phone number that will be displayed to the user when the flashcall is received on the
+   * The phone number that will be displayed to the user when the phone call is received on the
```

Comment:
The comments need to be reverted in this file too.

Reply:
will fix OAS too